File: app/src/main/java/com/bitsanddroids/myplants/plants/PersonalPlant.java
package com.bitsanddroids.myplants.plants;

import lombok.Data;
import java.util.Date;

@Data
public class PersonalPlant {

    private String name;
    private String ImageUrl;
    private int sun;
    private int placement;
    private int toxic;
    private int water;
    private int edible;
    private Date watered;
    private String plantID;
    private String placementNote;

    private String userID;

    public PersonalPlant(){

    }

    public PersonalPlant(String name, String imageUrl, int sun, int placement, int toxic, int water, int edible, String userID) {
        this.name = name;
        this.ImageUrl = imageUrl;
        this.sun = sun;
        this.placement = placement;
        this.toxic = toxic;
        this.water = water;
        this.edible = edible;
        this.userID = userID;
    }
}

File: app/src/main/java/com/bitsanddroids/myplants/plants/PersonalPlant.kt
package com.bitsanddroids.myplants.plants


import java.io.Serializable
import java.util.Date

class PersonalPlant(val name: String = ""
                    , val imageUrl: String = ""
                    , val sun: Int = 1
                    , val placement: Int = 1
                    , val toxic: Int = 1
                    , val water: Int = 1
                    , val edible: Int = 1
                    , var watered: Date? = null
                    , val plantID: String? = null
                    , var placementNote: String? = ""
                    , val plantingTime: String? = ""
                    , val userID: String? = "") : Serializable {

}
